docs/source/tests.rst

```diff
@@ -35,3 +35,119 @@ If you are adding tests or altering functionality, it might be faster to only ru
+  Occasionally we have also used, in few tests, the default address in ``neo-test-coz.wallet``. The coz wallet is special as it is the original coz privnet wallet with 100000000 NEO. 
+
+- The tests which depend directly on  ``WalletFixtureTestCase`` are generated using transactions between addresses contained in W wallets (``neo-test1-w.wallet``, ``neo-test2-w.wallet``, ``neo-test3-w.wallet``).
+  
+ A picture may help to visualize the tests BC and W:
+
+.. image:: images/fixture-tests.PNG
+
+When adding new tests to the privnet fixtures try to follow the following guidelines in order to minimize the dependencies between unit-tests:
+ 
+a) We should strive to not use directly the ``neo-test-coz.wallet`` in any tests as it is expected to be the source of NEO and GAS for other test wallet and it is natural that it will change quite a bit in due course of time. 
+   If we create tests which involve the ``neo-test-coz.wallet``, then we should be ready to update them frequently. 
+   This is not good from a maintenance perspective.
+b) Try to re-use existing transactions on the privnet fixture for creating new tests.
+c) Add new transactions to the privnet fixture only when required as this will add the risk to break existing tests. 
+   We are aware that we have a couple of assertions related tot the blockchain‘s height and wallet‘s height. 
+   These tests inevitably need to be updated after adding new transactions to the privnet fixtures.
+d) Create your own token if you want to be sure to isolate your tests.
+e) Create new wallets only if absolutely necessary. 
+
+**The last three points c), d), e) required updating the fixtures:**
+
+1) Pull the lastet ``neo-privnet-unittest`` image: (`link <https://hub.docker.com/r/dautt/neo-privnet-unittest/>`_)
+
+.. code-block:: sh
+
+	docker pull dautt/neo-privnet-unittest:v0.0.xx
+
+
+2) Run the image:
+
+.. code-block:: sh
+	
+	docker run --rm -d --name neo-privnet-unittest -p 20333-20336:20333-20336/tcp -p 30333-30336:30333-30336/tcp dautt/neo-privnet-unittest:v0.0.xx``
+
+3) Clean the current ``unittest`` chain:
+
+.. code-block:: sh
+	
+	rm -rf ~/.neopython/Chains/unittest
+	rm -rf ~/.neopython/Chains/unittest_notif
+
+4) Activate your virtual environment:
+
+.. code-block:: sh
+	
+	source venv/bin/activate
+
+5) Start the neo node:
+
+.. code-block:: sh
+	
+	python prompt.py -u
+
+6) Use the following wallets for generating transactions:
+
+.. code-block:: sh
+
+	neo-test-coz.wallet	(pwd = coz)							
+	neo-test1-bc.wallet	(pwd = 1234567890) 
+	neo-test2-bc.wallet	(pwd = 1234567890)                                                                                                                   	                  
+	neo-test1-w.wallet	(pwd = 1234567890)						
+	neo-test2-w.wallet	(pwd = 1234567890) 
+	neo-test3-w.wallet	(pwd = 1234567890)
+	
+(If you want to add new tests (BC) then use BC wallets. Otherwise use W wallets for new tests (W))
```

Comment:
This line is kind of hard to understand. If you read it carefully it says:
If I want to add new tests, use BC wallets. (Otherwise == )If you _do not_  want to add new test, then use W wallets for new tests. 

I think we need to make it clearer when to choose which wallet. When you do so, can you please refer to the wallets as
> use the `-bc.wallet` wallets
> use the `-w.wallet` wallets

that makes it a bit more obvious for new readers that the BC/W abbreviation is part of the file name.

Reply:
You marked this as resolved but I don't think it is fully addressed. The text is nearly identical besides changing 
> then use BC wallets.

to

> then use the `-bc-wallet` wallets

That change is good but only 1 part of the request. The second part is that the description is still ambiguous and we need to make a clear description for users such that they how to decide between using `-bc.wallet` wallets and when to use `-w.wallet` wallets. 
Please also drop the `(BC)` and `(W)` as they have no added value when we already refer to `-bc.wallet` and `-w.wallet` (which have a more direct reference to the files). Thanks!

Reply:
Okay, I have changed the ambiguous sentence so that it is super clear. Let us hope it works fine now. :smile: